src/esp/gfx/LightSetup.cpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+#include "LightSetup.h"
+
+namespace esp {
+namespace gfx {
+
+Magnum::Vector3 getLightPositionRelativeToCamera(
+    const LightInfo& light,
+    const Magnum::Matrix4& transformationMatrix,
+    const Magnum::Matrix4& cameraMatrix) {
+  if (light.model == LightPositionModel::OBJECT)
+    return transformationMatrix.transformPoint(light.position);
+
+  if (light.model == LightPositionModel::GLOBAL)
+    return cameraMatrix.transformPoint(light.position);
+
+  // LightPositionModel::CAMERA
```

Comment:
`CORRADE_INTERNAL_ASSERT()` this? Would ensure this function doesn't get forgotten when adding new light position models.